packages/react-data-grid/src/masks/InteractionMasks.js

```diff
@@ -29,7 +29,7 @@ import keyCodes from '../KeyCodes';
 const SCROLL_CELL_BUFFER = 2;
 
 export default class InteractionMasks extends React.Component {
-  static dispplayName = 'InteractionMasks';
+  static displayName = 'InteractionMasks';
```

Comment:
do we really need this `displayName`?

Reply:
It's not a necessity, but it helps with debugging as class/function names will change with minification.